src/design/demo/favicon-svg.twig

```diff
@@ -0,0 +1,15 @@
+<style>
```

Comment:
Instead of including these with an inline style, can we include a `demo/styles.scss` file similar to other demo-specific styles?